src/coreclr/gc/objecthandle.cpp

```diff
@@ -1602,7 +1611,9 @@ void Ref_UpdatePointers(uint32_t condemned, uint32_t maxgen, ScanContext* sc, Re
 #ifdef FEATURE_WEAK_NATIVE_COM_HANDLES
         HNDTYPE_WEAK_NATIVE_COM,
 #endif
+#ifdef FEATURE_SIZED_REF_HANDLES
         HNDTYPE_SIZEDREF,
+#endif
```

Comment:
```suggestion
#endif // FEATURE_SIZED_REF_HANDLES
```

I think we're trying to be consistent about the ending comment for ifdefs.